res/res_agi.c

```diff
@@ -2415,6 +2415,8 @@ static int handle_recordfile(struct ast_channel *chan, AGI *agi, int argc, const
 			}
 			switch(f->frametype) {
 			case AST_FRAME_DTMF:
+				ast_debug(3, "DUB: no processing for DTMF digit=%d, flag=%d \n", f->subclass.integer,
```

Comment:
Can you explain this scenario.  What is the debug statement saying;  why no processing for DTMF?